config/routes.rb

```diff
@@ -34,5 +34,7 @@
     to: "services_and_information#index",
     as: :services_and_information
 
+  get '/learn-to-drive-a-car', to: 'tasklist#show'
```

Comment:
This will be a duplicate of https://github.com/alphagov/collections/pull/400/files#diff-21497849d8f00507c9c8dcaf6288b136R9

The reason is that I need it here to write the path.